tests/components/mealie/snapshots/test_diagnostics.ambr

```diff
@@ -516,12 +516,12 @@
             'food_id': '96801494-4e26-4148-849a-8155deb76327',
             'is_food': True,
             'item_id': '69913b9a-7c75-4935-abec-297cf7483f88',
-            'label_id': None,
+            'label_id': 'None',
```

Comment:
@joostlek @andrew-codechimp with new serialization of mashumaro, it guarantees the data is written in the type defined on the class. Looks like `label_id` is typed to not be able to be None, but the integration passes `None`. Same for `unit_id`.